src/pages/LoginPage.tsx

```diff
@@ -32,41 +32,46 @@ const emailInvalidationRules = [
 
 const passwordRegex = new RegExp("^[-/=!#$%&'*+?^_`{|}~.A-Z0-9]{" + PASSWORD.minLength + "," + PASSWORD.maxLength + "}$", "i");
 
-const validate = async (values: FormValues) => {
-  let errors: FormErrors = {};
-  if (emailInvalidationRules.some(rule => rule.test(values.email))) {
-    errors.email = 'Please enter a valid email address';
-  }
-  if (!passwordRegex.test(values.password)) {
-    errors.password = `An invalid character is present in the password. Password must be between ${PASSWORD.minLength} and ${PASSWORD.maxLength} characters; upper or lower case Latin letters (a–z, A–Z); numbers from 0 to 9; symbols ! # $ % & ' * + - / = ? ^ _ \` { | } ~`;
-  }
-  if (values.password.length >= PASSWORD.maxLength || values.password.length < PASSWORD.minLength) {
-    errors.password = `Password must be between ${PASSWORD.minLength} and ${PASSWORD.maxLength} characters`;
-  }
-  return errors;
-}
-
 const LoginPage = () => {
   const [isChecked, setIsChecked] = useState(false);
+  let loading = useAppSelector(state => state.login.loading);
 
   const dispatch = useAppDispatch();
 
   const initialValues: FormValues = {
     email: '',
     password: '',
     remember: isChecked,
-  }
+  };
 
   return (
     <div className='field'>
+      {loading &&
+        <h1 style={{position: 'fixed', left: '50%', transform: 'translate(-50%, -40%)', color: 'grey'}}>Loading
+          ...</h1>}
       <Formik
         initialValues={initialValues}
-        validate={validate}
-        onSubmit={(values: object) => {
-          dispatch(getLogin(values))
+        validate={async (values: FormValues) => {
+          let errors: FormErrors = {};
+          if (emailInvalidationRules.some(rule => rule.test(values.email))) {
+            errors.email = 'Please enter a valid email address';
+          }
+          if (!passwordRegex.test(values.password)) {
+            errors.password = `An invalid character is present in the password. Password must be between ${PASSWORD.minLength} and ${PASSWORD.maxLength} characters; upper or lower case Latin letters (a–z, A–Z); numbers from 0 to 9; symbols ! # $ % & ' * + - / = ? ^ _ \` { | } ~`;
+          }
+          if (values.password.length >= PASSWORD.maxLength || values.password.length < PASSWORD.minLength) {
```

Comment:
'values.password.length >= PASSWORD.maxLength'

includes the case when 
password length = PASSWORD.maxLength

in this case you do not need error, please replace with

 (values.password.length > PASSWORD.maxLength || values.password.length < PASSWORD.minLength)